.github/workflows/release.yml

```
@@ -104,6 +104,9 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
+      - uses: actions/checkout@v4
+        with:
+          persist-credentials: false
```

Mind adding an empty line just for consistency?

```suggestion
          persist-credentials: false

```